Mac builds failing

```
+ opam config exec -- make -s -j4 STATICLINK=1 'LIB_PARAMS=/usr/local/lib/libz.a /usr/local/lib/libpcre2-8.a /usr/local/lib/libmbedtls.a /usr/local/lib/libmbedcrypto.a /usr/local/lib/libmbedx509.a -cclib '\''-framework Security -framework CoreFoundation'\''' haxe
Warning:  exec was deprecated in version 2.1 of the opam CLI. Use opam exec instead or set OPAMCLI environment variable to 2.0.
File "libs/mbedtls/dune", line 7, characters 9-22:
7 | 		(names mbedtls_stubs))
             ^^^^^^^^^^^^^
(cd _build/default/libs/mbedtls && /usr/bin/cc -O2 -fno-strict-aliasing -fwrapv -D_FILE_OFFSET_BITS=64 -D_REENTRANT -g -I /Users/runner/.opam/4.08.1/lib/ocaml -o mbedtls_stubs.o -c mbedtls_stubs.c)
mbedtls_stubs.c:21:10: fatal error: 'mbedtls/debug.h' file not found
#include "mbedtls/debug.h"
         ^~~~~~~~~~~~~~~~~
1 error generated.
make: *** [haxe] Error 1
Error: Process completed with exit code 2.
```

Doesn't look like it's just random unfortunately.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -617,7 +617,7 @@ jobs:
         env:
           # For compatibility with macOS 10.13
           ZLIB_VERSION: 1.3
-          MBEDTLS_VERSION: 2.25.0
+          MBEDTLS_VERSION: 2.28.5
           PCRE2_VERSION: 10.42
         run: |
           set -ex

--- a/extra/github-actions/build-mac.yml
+++ b/extra/github-actions/build-mac.yml
@@ -2,7 +2,7 @@
   env:
     # For compatibility with macOS 10.13
     ZLIB_VERSION: 1.3
-    MBEDTLS_VERSION: 2.25.0
+    MBEDTLS_VERSION: 2.28.5
     PCRE2_VERSION: 10.42
   run: |
     set -ex

